github3/repos/branch.py

```diff
@@ -28,5 +29,51 @@ def _update_attributes(self, branch):
         #: Provides the branch's protection status.
         self.protection = branch.get('protection')
 
+        if 'self' in self.links:
+            self._api = self.links['self']
```

Comment:
This is sad. :( I'll ping them about including a `url` attribute like other resources do and see if it's plausible.